quipucords/api/source/serializer.py

```diff
@@ -77,6 +77,13 @@ class SourceSerializer(NotEmptySerializer):
     options = SourceOptionsSerializer(required=False, many=False)
     credentials = CredentialsField(many=True, queryset=Credential.objects.all())
 
+    HTTP_SOURCE_TYPES = (
+        DataSources.ANSIBLE,
+        DataSources.OPENSHIFT,
+        DataSources.SATELLITE,
+        DataSources.VCENTER,
+    )
```

Comment:
how come DataSources.NETWORK is not here ?

Reply:
DataSources.NETWORK relies on the logic for "ssh_source", which is not part of the deduplication. Those other sources are kinda the same thing when we think about how their source model should behave

Reply:
we could have a `SSH_SOURCE_TYPES` constant, which would only contain `DataSources.NETWORK` 🤔